app/src/main/java/org/wikipedia/watchlist/WatchlistItemView.kt

```diff
@@ -52,17 +54,50 @@ class WatchlistItemView constructor(context: Context, attrs: AttributeSet? = nul
         userNameText.text = item.user
 
         userNameText.setIconResource(if (item.isAnon) R.drawable.ic_anonymous_ooui else R.drawable.ic_user_talk)
-
-        val diffByteCount = item.newlen - item.oldlen
-        if (diffByteCount >= 0) {
-            diffText.setTextColor(if (diffByteCount > 0) ContextCompat.getColor(context, R.color.green50) else ResourceUtil.getThemedColor(context, R.attr.material_theme_secondary_color))
-            diffText.text = String.format("%+d", diffByteCount)
+        if (item.logType.isNotEmpty()) {
+            when (item.logType) {
+                context.getString(R.string.page_moved) -> {
+                    setButtonTextAndIconColor(context.getString(R.string.watchlist_page_moved),
+                            ResourceUtil.getThemedColor(context, R.attr.color_group_61),
+                            ContextCompat.getDrawable(context, R.drawable.ic_info_outline_black_24dp),
+                            R.attr.suggestions_background_color)
```

Comment:
Would it be possible to have the same way of sending attributes and let the method does the job?

```
R.attr.color_group_61,
R.drawable.ic_info_outline_black_24dp,
R.attr.suggestions_background_color)
```

Reply:
I did it this way because we will have a chance to send null when we dont want an icon.. is it ok to have it this way so we dont need to make a null check inside the function?

Reply:
Please see the follow-up PR.